auth-web/src/components/auth/staff/continuation-application/ExtraprovincialRegistrationBc.vue

```diff
@@ -72,32 +72,54 @@
 </template>
 
 <script lang="ts">
-import { Component, Prop, Vue } from 'vue-property-decorator'
-import { ContinuationReviewFilingIF, ContinuationReviewIF } from '@/models/continuation-review'
-import DateUtils from '@/util/date-utils'
+import { ContinuationFilingIF, ContinuationReviewIF } from '@/models/continuation-review'
+import { defineComponent, reactive } from '@vue/composition-api'
+import CommonUtils from '@/util/common-util'
+import moment from 'moment'
 
-@Component({})
-export default class ExtraprovincialRegistrationBc extends Vue {
-  /** Continuation Review object that comes from parent component. */
-  @Prop({ required: true }) readonly continuationReview: ContinuationReviewIF
+export default defineComponent({
+  name: 'ExtraprovincialRegistrationBc',
 
-  get continuationIn (): ContinuationReviewFilingIF {
-    return this.continuationReview?.filing?.continuationIn
-  }
+  props: {
+    review: { type: Object as () => ContinuationReviewIF, required: true },
+    filing: { type: Object as () => ContinuationFilingIF, required: true }
+  },
 
-  get identifier (): string {
-    return this.continuationIn?.business?.identifier
-  }
+  setup (props) {
+    const state = reactive({
+      get continuationIn (): any {
+        return props.filing?.continuationIn
+      },
```

Comment:
I found that I could use this "getter" syntax instead of "computed", as long as they reference other state property using "this" (eg, line 95).

Travis, are you OK with this?

Reply:
Shouldn't be using `this` composition-api was trying to get away from the `this` context

Reply:
OK. Fixing it now.

Reply:
Fixed in latest commit. Resolving